ballerina/annotation_types.bal

```diff
@@ -46,11 +45,20 @@ public annotation AutoIncrementConfig AutoIncrement on record field;
 # + reference - The names of the fields of the other entity, which are referenced by the `keyColumns`
 # + onDelete - The action to be taken when the referenced value in the parent entity is deleted
 # + onUpdate - The action to be taken when the referenced value in the parent entity is updated
+# + joiningTable - The details of the joining table to be used in a many-to-many relation
 public type RelationConfig record {|
     string[] keyColumns?;
     string[] reference?;
     ReferenceAction onDelete?;
     ReferenceAction onUpdate?;
+    JoiningTable joiningTable?;
```

Comment:
```suggestion
    string joiningTable?;
```

Reply:
Fixed in [2db76bc](https://github.com/ballerina-platform/module-ballerina-persist/pull/82/commits/2db76bcb4776b1a51558bd4bd1d9b5f5e143213f)